src/frameworks/flask/index.ts

```diff
@@ -0,0 +1,79 @@
+import { copy, mkdirp } from "fs-extra";
+import { mkdir, readFile, readdir, writeFile } from "fs/promises";
+import { join, relative } from "path";
+import { BuildResult, FrameworkType, SupportLevel } from "../interfaces";
+import { dirExistsSync } from "../../fsutils";
+import { findPythonCLI, hasPipDependency, spawnPython } from "../utils";
+import { spawnSync } from "child_process";
```

Comment:
spawn should come from cross-spawn to prevent issues with Windows

```suggestion
import { sync as spawnSync } from "cross-spawn";
```